packages/cli/src/validators/createTableDefOverriderList.ts

```diff
@@ -0,0 +1,36 @@
+import { ColumnDefProperties } from '@synthql/queries';
+import { SchemaDefOverrides } from '../types/CliConfig';
+
+interface TableDefOverrider {
+    isValidQualifiedTableName: (qualifiedTableName: string) => boolean;
+    applyColumnDefOverride: (
```

Comment:
This signature doesn't make sense. This should take an existing `ColumnDefProperties` as input and just transform it.